xds/src/test/java/io/grpc/xds/ClientXdsClientTestBase.java

```diff
@@ -247,48 +298,37 @@ public void ldsResourceNotFound() {
 
   @Test
   public void ldsResourceFound_containsVirtualHosts() {
-    DiscoveryRpcCall call =
-        startResourceWatcher(ResourceType.LDS, LDS_RESOURCE, ldsResourceWatcher);
-    List<Any> listeners = ImmutableList.of(
-        Any.pack(mf.buildListener(LDS_RESOURCE,
-            mf.buildRouteConfiguration("do not care", mf.buildOpaqueVirtualHosts(2)))));
-    call.sendResponse("0", listeners, ResourceType.LDS, "0000");
+    DiscoveryRpcCall call = startResourceWatcher(LDS, LDS_RESOURCE, ldsResourceWatcher);
+    fakeClock.forwardTime(10, TimeUnit.SECONDS);
```

Comment:
Why do we need this time? It should be irrelevant to what the test is trying to assert.

Reply:
Good 👀 . To be honest, I don't know where it came from. I think I was looking up something and forgot to delete it 🤷